lib-es5/utils/index.js

```diff
@@ -148,7 +148,10 @@ function textStyle(layer) {
  * @return {Object|String} A normalized String of the input value if possible otherwise the value itself
  */
 function normalize_expression(expression) {
-  if (!isString(expression) || expression.length === 0 || expression.match(/^!.+!$/)) {
+  // Exclude(Don't normalize) expressions that are variable strings foo_!width!
+  // Exclude(Don't normalize) expressions that contain a semicolon (e_preview:duration)
```

Comment:
An expression candidate (can be a complex string) can have a ":", I would not exclude those.
I would use a look behind approach like in the reference implementation.

Reply:
Fixed